docs/guides/ipsec-guide.md

```diff
@@ -0,0 +1,129 @@
+# IPsec Offload Guide
+
+## 1. Overview
+
+The [ipdk-io/ipsec-recipe](https://github.com/ipdk-io/ipsec-recipe) is an
```

Comment:
Your first sentence (or paragraph, or few paragraphs) should tell the reader what the feature does for them. Don't start out talking about recipes or strongSwan plugin code. Those are details.